charts/prometheus-adapter/templates/custom-metrics-cluster-role.yaml

```diff
@@ -12,6 +12,6 @@ metadata:
 rules:
 - apiGroups:
   - custom.metrics.k8s.io
-  resources: ["*"]
+  resources: {{ toYaml .Values.rbac.customMetricsResources | nindent 2 }}
```

Comment:
Would prefer `rbac.customMetrics.resources` in case someone wants to customize `verbs` in the future.